flow/environment.py

```diff
@@ -392,28 +392,43 @@ class SimpleSchedulerEnvironment(ComputeEnvironment):
     scheduler_type = SimpleScheduler
     template = "simple_scheduler.sh"
 
-
+@deprecated(
+    deprecated_in="0.16",
+    removed_in="0.17",
```

Comment:
Our next release is 0.14, so that's when the deprecation period should begin. I think it's fine to deprecate this in one minor release, according to our deprecation policy: https://docs.signac.io/projects/flow/en/latest/support.html#deprecation-policy
```suggestion
    deprecated_in="0.14",
    removed_in="0.15",
```